### PR TITLE
 Add workflow to publish to pypi(test)

### PR DESCRIPTION
**Pull request by cgtobi**
This requires GitHub secrets for pypi (test): `PYPI_PROD_TOKEN` and `PYPI_TEST_TOKEN`

### CODE REVIEW

**Comment by jabesq** [comment]
@dependabot rebase

File: .github/workflows/publish-to-test-pypi.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  create:
+    tags:
+      - '*'
+
+jobs:
+  build-n-publish:
+    name: Build and publish 📦 to TestPyPI
+    runs-on: ubuntu-latest
+    steps:
+<<<<<<< HEAD
```
Comment (jabesq) [inline .github/workflows/publish-to-test-pypi.yml]: You committed some unresolved merge conflict

File: src/pyatmo/version.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+MAJOR_VERSION = 4
```
Comment (jabesq) [inline src/pyatmo/version.py]: I created the same file in src. I don't think that we need to put it inside pyatmo directory

File: .github/workflows/publish-to-test-pypi.yml (hunk 1 above)
Comment (cgtobi) [reply .github/workflows/publish-to-test-pypi.yml]: Oops, let me fix that.

**Review by jabesq — APPROVED** [review]

**Pull request closed without merging**